internal/receiver/databricksreceiver/metadata.yaml

```diff
@@ -62,3 +94,1118 @@ metrics:
       value_type: int
     attributes:
       [job_id, task_id]
+
+  databricks.spark.codegenerator.compilationtime:
```

Comment:
what about snake casing instead of concatenation?

```suggestion
  databricks.spark.code_generator.compilation_time:
```

Reply:
Fixed (there were a lot -- previous naming came directly from Spark).